applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -303,7 +303,6 @@ def ValidateSettings(self):
             self.settings["fm_ale_settings"]["fm_ale_solver_settings"].ValidateAndAssignDefaults(self._get_fm_ale_solver_default_settings(mesh_movement))
 
     def __init__(self, model, custom_settings):
-        self._validate_settings_in_baseclass=True # To be removed eventually
```

Comment:
Are we removing this now? Finally :) :+1:

Reply:
Actually, I think it hasn't been used for a while. :smile: